Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -69,9 +103,58 @@ public final class ProjectPamphletViewController: UIViewController {
     }
   }
 
+  private func configureViews() {
+    // Configure subviews
+    self.backThisProjectContainerView.addSubview(self.backThisProjectButton)
+
+    self.view.addSubview(self.backThisProjectContainerView)
+    self.view.bringSubviewToFront(backThisProjectContainerView)
+
+    self.backThisProjectButton.addTarget(self, action: #selector(backThisProjectTapped), for: .touchUpInside)
+
+    // Configure constraints
+    // swiftlint:disable line_length
```

Comment:
I would just indent these like this then we don't have to disable `line_length` rule. Also disabling it in this way disables it for the rest of the file.

```swift
    let backThisProjectContainerViewConstraints = [
      self.backThisProjectContainerView.leftAnchor.constraint(equalTo: self.view.leftAnchor),
      self.backThisProjectContainerView.rightAnchor.constraint(equalTo: self.view.rightAnchor),
      self.backThisProjectContainerView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor)
    ]
    
    let containerMargins = self.backThisProjectContainerView.layoutMarginsGuide

    let backThisProjectButtonConstraints = [
      self.backThisProjectButton.leftAnchor.constraint(equalTo: containerMargins.leftAnchor),
      self.backThisProjectButton.rightAnchor.constraint(equalTo: containerMargins.rightAnchor),
      self.backThisProjectButton.bottomAnchor.constraint(equalTo: containerMargins.bottomAnchor),
      self.backThisProjectButton.topAnchor.constraint(equalTo: containerMargins.topAnchor),
      self.backThisProjectButton.heightAnchor.constraint(equalToConstant: Styles.buttonMinHeight)
    ]
```